io.github.hakuneko.HakuNeko.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+	<screenshots>
+   	 <screenshot type="default">
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-pages.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-video.png</image>
+    </screenshot>
+	<screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-reader.png</image>
+    </screenshot>
+	<screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-websites.png</image>
+    </screenshot>
+	<screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-settings.png</image>
+    </screenshot>
+  	</screenshots>
+	<releases>
+    	<release version="v6.1.7" date="2020-01-18"/>
+	</releases>
+	<content_rating type="oars-1.0">
+		<content_attribute id="drugs-alcohol">intense</content_attribute>
+    	<content_attribute id="drugs-narcotics">intense</content_attribute>
+    	<content_attribute id="drugs-tobacco">intense</content_attribute>
+    	<content_attribute id="language-discrimination">intense</content_attribute>
+    	<content_attribute id="language-humor">intense</content_attribute>
+    	<content_attribute id="language-profanity">intense</content_attribute>
+    	<content_attribute id="money-gambling">intense</content_attribute>
+    	<content_attribute id="money-purchasing">intense</content_attribute>
+    	<content_attribute id="sex-adultery">intense</content_attribute>
+    	<content_attribute id="sex-appearance">intense</content_attribute>
+    	<content_attribute id="sex-homosexuality">intense</content_attribute>
+    	<content_attribute id="sex-nudity">intense</content_attribute>
+    	<content_attribute id="sex-prostitution">intense</content_attribute>
+    	<content_attribute id="sex-themes">intense</content_attribute>
+    	<content_attribute id="social-audio">intense</content_attribute>
+    	<content_attribute id="social-chat">intense</content_attribute>
+    	<content_attribute id="social-contacts">intense</content_attribute>
+    	<content_attribute id="social-info">intense</content_attribute>
+    	<content_attribute id="social-location">intense</content_attribute>
+    	<content_attribute id="violence-bloodshed">intense</content_attribute>
+    	<content_attribute id="violence-cartoon">intense</content_attribute>
+    	<content_attribute id="violence-desecration">intense</content_attribute>
+    	<content_attribute id="violence-fantasy">intense</content_attribute>
+    	<content_attribute id="violence-realistic">intense</content_attribute>
+    	<content_attribute id="violence-sexual">intense</content_attribute>
+    	<content_attribute id="violence-slavery">intense</content_attribute>
+    	<content_attribute id="violence-worship">intense</content_attribute>
+	</content_rating>
+	<update_contact>https://discord.gg/A5d3NDf</update_contact>
```

Comment:
Update contact should be an email.

Reply:
I need to ask @ronny1982 for an email as I didn't see one on the project site.

Reply:
What is "update_contact" and what is the supposed role (source code maintainer, package container, ...)?
The software is UNLICENSED and therefore belongs to the community.

Reply:
Someone with write access to upstream repo.

Reply:
I am actually not sure what to put in the `update_contact` tag

Reply:
@ronny1982 Maybe we could have an email address that reflects the HakuNeko development team?

Reply:
I have fixed the `update_contact` tag.